src/components/RecipeContainer.js

```diff
@@ -1,23 +1,29 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import Ingredients from './Ingredients';
 import Directions from './Directions';
 import sampleRecipes from '../sample-recipe';
 
-class RecipeContainer extends React.Component {
-  render() {
-    const recipeId = this.props.match.params.recipeId;
-    const recipe = sampleRecipes.find((recipe) => recipe.url == recipeId);
+const RecipeContainer = (props) =>{
+    const recipeId = props.match.params.recipeId;
 
-    return (
-      <div className="pageContent">
-        <Ingredients ingredientsList={recipe.ingredients}/>
-        <div className="recipeContainer">
-          <h2>{recipe.name}</h2>
-          <Directions directionsList={recipe.directions}/>
-        </div>
+    const [recipe, singleRecipe] = useState({});
```

Comment:
By convention this should be `set...` so if you want to keep `singleRecipe` you can change it to `setSingleRecipe`.
Or, since the recipe variable is called `recipe` not `singleRecipe`, you can rename `singleRecipe` to `setRecipe`